pkg/remote/azurerm/repository/storage.go

```diff
@@ -0,0 +1,98 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/armcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/armstorage"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/cloudskiff/driftctl/pkg/remote/azurerm/common"
+	"github.com/cloudskiff/driftctl/pkg/remote/cache"
+)
+
+type StorageRespository interface {
+	ListAllStorageAccount() ([]*armstorage.StorageAccount, error)
+	ListAllStorageContainer(account *armstorage.StorageAccount) ([]string, error)
+}
+
+// interfaces below are only used to be able to create mock on azure sdk
+type blobContainerClient interface {
+	List(resourceGroupName string, accountName string, options *armstorage.BlobContainersListOptions) armstorage.ListContainerItemsPager
+}
```

Comment:
If it's only useful for mocks, shouldn't it be located in `test/azure/storage.go` ?